### ISSUE TITLE
i18n: Text string "Select the trace"

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/ios#1956]
@marinofaggiana Can you please explain what the following text string is for. What is a "trace"?

https://github.com/nextcloud/ios/blob/83c1d61ed3e1f76decd6f6eb76b70308d84343be/iOSClient/Supporting%20Files/el.lproj/Localizable.strings#L835

_Reported in Transifex by the translator._

**Comment by marinofaggiana** [comment]
trace = trace video or trace audio

**Comment by marinofaggiana** [comment]
No, for example if I have a music CD with 10 music I select the trace N. 7 for example or a Video with several trace (example MKV) i can select the internal trace.

The "Select the trace" is perfect.

**Comment by Valdnet** [comment]
Thanks for the clarification 😉.

**Comment by Valdnet** [comment]
@marinofaggiana Translators are asking if there should be "trace" -> "track"?
In my opinion, it should be "track" too, but I don't know if it is compatible with the function.

**Comment by rakekniven** [comment]
Is is something about internal video format?

If it is about audio it should be "Select track". Never heard "trace" in relation to audio.
